maquetado/scss/styles.scss

```diff
@@ -3,20 +3,21 @@
 }
 
 .book-container {
-  display: flex;
-  flex-direction: column;
   padding: 10px;
 }
 
-.book-title {
-  font-size: 32px;
-  margin: 20px 0 40px;
+.book-cover {
+  height: 100%;
+  max-width: 380px;
+  margin-right: 40px;
+  width: 100%;
+  max-width: 260px;
```

Comment:
You have two `max-width` here. this is only use for width for responsive purposes. In other words the screen is just going to change horizontally and elements need to be re arranged like that. Vertically the user will just scroll down in smaller screens. 

```
height: 380px;
max-width: 260px;
width: 100%; 
```

Remember the alphabetical order

Reply:
fixed, bad typo